libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationEventDispatcher.java

```diff
@@ -166,6 +187,12 @@ void onFasterRouteEvent(DirectionsRoute directionsRoute) {
     }
   }
 
+  void onLocationUpdate(Location location) {
+    for (RawLocationListener listener : rawLocationListeners) {
+      listener.onLocationUpdate(location);
```

Comment:
Is there a use case for multiple listeners? I worry that for all these listeners, we're adding this logic unnecessarily

Reply:
I added the ability to add multiple listeners to align with the other listeners in `MapboxNavigation`.